src/components/DownloadImageDialog/VariantSelector.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+	Box,
+	FormControl,
+	FormControlLabel,
+	FormLabel,
+	Radio,
+	RadioGroup,
+	Tooltip,
+	Typography,
+} from '@mui/material';
+import { MUILinkWithTracking } from '../MUILinkWithTracking';
+import { getOsVariantDisplayText } from './utils';
+import { VersionSelectionOptions } from './version';
+import { Lightbulb } from '@mui/icons-material';
+
+const BuildVariants = ['dev', 'prod'] as const;
+export type BuildVariant = (typeof BuildVariants)[number];
+interface VaraintSelectorProps {
+	version: VersionSelectionOptions | undefined;
+	onVariantChange: (isDev: boolean) => void;
+	variant: BuildVariant;
+}
+
+export const VariantSelector: React.FC<VaraintSelectorProps> = ({
+	version,
+	variant,
+	onVariantChange,
+}) => {
+	return (
+		<FormControl>
+			<FormLabel>Select edition</FormLabel>
+			<RadioGroup
+				aria-labelledby="variant-radio-buttons-group"
+				name="developmentMode"
+				key="varian"
+				value={variant === 'dev'}
+				onChange={(event) => {
+					onVariantChange(event.target.value === 'true');
+				}}
+			>
+				{BuildVariants.map((buildVariant, index) => {
+					const label = (
+						<Typography>{getOsVariantDisplayText(buildVariant)}</Typography>
+					);
+					const isDev = buildVariant === 'dev';
+					const isDisabled =
+						version == null ||
+						(version.hasPrebuiltVariants && !version.rawVersions[buildVariant]);
+					return (
+						<Tooltip
+							title={
+								isDisabled
+									? 'This edition is not available for the selected version'
+									: undefined
+							}
+							key={index}
+						>
+							<Box display="flex" flexDirection="column">
+								<FormControlLabel
+									sx={{ opacity: isDisabled ? 0.6 : 1 }}
+									disabled={isDisabled}
+									value={isDev}
+									control={<Radio />}
+									label={
+										isDev ? (
+											<Box
+												display="flex"
+												gap={1}
+												flexDirection="row"
+												alignItems="center"
+											>
+												<Typography>{label}</Typography>
+												<Typography
+													variant="smallText"
+													color="customBlue.main"
+													alignItems="center"
+													display="flex"
+												>
+													<Lightbulb sx={{ width: 14, height: 14 }} />
+													Recommended for first time users
+												</Typography>
+											</Box>
+										) : (
+											label
+										)
+									}
+								/>
+								{isDev ? (
```

Comment:
We should handle this with a dictionary by passing a "title" with "description" instead of needing a ternary to manually check which description to show depending on `isDev`